build.sbt

```diff
@@ -68,7 +70,7 @@ lazy val openLegalReviewReport =
     "report in review mode in the browser."
   )
 openLegalReviewReport := {
-  gatherLicenses.value
+  val _ = gatherLicenses.value
```

Comment:
why?

Reply:
Because sbt was complaining that it is unassigned.

I will re-check that.

Reply:
It is not complaining now, I don't know why it did before, maybe it was some error.

I will revert that change.